tests/Avalonia.IntegrationTests.Appium/readme.md

```diff
@@ -27,4 +27,11 @@
 
 Each time you make a change to Avalonia or `IntegrationTestApp`, re-run the `bundle.sh` script (registration only needs to be done once).
 
-
+### Appium 2
+
+Tests in this project are configured to run with Appium 1 (as only this version supports WinAppDriver).
+If you need to run with Appium 2 on macOS, extra steps are required:
+- Install Appium 2 with [mac2 driver](https://github.com/appium/appium-mac2-driver)
```

Comment:
maybe we can include the ff as well:

1.) install brew
2.) brew install appium
3) appium driver install mac2

Reply:
I would trust appium website with that instead

Reply:
yeah it was not on the readmes there but well